### DISCUSSION TITLE
Add an unranked toggle for HR in osu!catch to stop the random moving of notes.

### DISCUSSION THREAD

**Discussion by Wrongwayscout** [ppy/osu#26115] [Ideas]
I know this is a pretty key part of HR in CTB, but it would be interesting to be able to use HR and see maps without the random moving of notes. Unranked obviously since it changes gameplay drastically.

**Comment by Rekunan** [comment]
The end goal is to have every mod be rankable (except auto). In the case of this option, I could see there being some adjustment for  score mult and pp calc to make it work out.

**Comment by cdwcgt** [comment]
if you just want to play with only high ar, you can use DA (difficulty adjust) than do not turn on spicy patterns
